src/status_im/ui/screens/qr_scanner/views.cljs

```diff
@@ -48,7 +48,7 @@
        {:on-press #(re-frame/dispatch [:qr-scanner.callback/scan-qr-code-cancel opts])}
        "Cancel"]
       [quo/button
-       {:on-press #(re-frame/dispatch [:qr-scanner.callback/scan-qr-code-success opts (when-let [text @text-value] (string/trim text))])}
+       {:on-press #(re-frame/dispatch [:qr-scanner.callback/scan-qr-code-success opts (when-let [text @text-value] (-> text string/trim (string/replace #"^Ethereum:" "ethereum:")))])}
```

Comment:
This is for `qr-test-view` only
it's annoying that iOS input capitalize the first char